blackbox-test/features/server/start.feature

```diff
@@ -60,8 +60,9 @@ Feature: server start task
     When I run `lc server start`
     Then it should fail
     And the output should contain one of the following:
-      |image library/somefakeimagethatdoesntexist:latest not found                        |
-      |repository somefakeimagethatdoesntexist not found: does not exist or no pull access|
+      |image library/somefakeimagethatdoesntexist:latest not found                                   |
+      |repository somefakeimagethatdoesntexist not found: does not exist or no pull access           |
+      |pull access denied for somefakeimagethatdoesntexist, repository does not exist or may require |
```

Comment:
The two previous error messages had a colon between the two clauses, but this one has a comma. Should it be a colon?

Reply:
the comma is in the new error message from docker 17.09:

```
$ docker pull notreal
Using default tag: latest
Error response from daemon: pull access denied for notreal, repository does not exist or may require 'docker login'
```

Reply:
😞